docs/configuration.md

```diff
@@ -26,6 +26,11 @@ connectivityCheck: true # whether you want to display a message when the apps ar
 # Optional theming
 theme: default # 'default' or one of the theme available in 'src/assets/themes'.
 
+# Optional custom stylesheet
+# Allows for @import statements and any other CSS. Useful for custom icon sets.
+# stylesheet: >-
+#   @import "assets/css/all.min.css";
```

Comment:
What about just: 
```yaml
stylesheet: "assets/css/customization.css"
```
\+ on line 164: 

```js
this.createStylesheet(`@import  ${this.config.stylesheet}`);
```

To avoid mixing concept in `yaml` file and keep it pure configuration. Somebody less familiar with css and yaml could be confused seeing the `@import` here.

Reply:
I agree. I was originally thinking that people could add simple styles here which would load faster than a separate css file, but I guess the `colors` config covers most of that anyways. Thanks for the feedback!